velox/type/tests/TypeTest.cpp

```diff
@@ -162,6 +162,23 @@ TEST(Type, DateComparison) {
   EXPECT_GE(epoch, beforeEpoch);
 }
 
+TEST(Type, ParseStringToDate) {
```

Comment:
naming: ParseStringToDate -> parseStringToDate + let's file an issue to refactor this file to match coding guidelines